node/src/Pos/Launcher/Resource.hs

```diff
@@ -203,22 +203,10 @@ bracketNodeResources :: forall ssc m a.
     -> Production a
 bracketNodeResources np sp k = bracketTransport tcpAddr $ \transport ->
     bracketKademlia (npBaseParams np) (npNetworkConfig np) $ \networkConfig ->
-    bracket (allocateNodeResources transport networkConfig np sp) releaseNodeResources $ \nodeRes -> do
-      case npInitDelay np of
-        Nothing ->
-          -- Start node immediately
-          return ()
-        Just initDelay -> do
-          now <- liftIO $ Time.getCurrentTime
-          let delay = (toUSec . toSec) (initDelay `Time.diffUTCTime` now)
-          if delay >= 0 then do
-            logNotice $ sformat ("Delaying node startup until " % shown) initDelay
-            liftIO $ threadDelay delay
-          else do
-            logWarning $ sformat ("--init-delay parameter in the past")
-      -- Notify systemd we are fully operative
-      notifyReady
-      k nodeRes
+        bracket (allocateNodeResources transport networkConfig np sp) releaseNodeResources $ \nodeRes ->do
```

Comment:
I am not entirely sure if we ended up removing the `--init-delay`, but I was under the impression @edsko reverted his work eventually, so I think GH is being over conservative here is showing as deletions something that, in practice, was already removed in `upstream/master`.